src/EPR.Calculator.Frontend/Controllers/LocalAuthorityDisposalCostsController.cs

```diff
@@ -7,7 +9,69 @@ public class LocalAuthorityDisposalCostsController : Controller
     {
         public IActionResult Index()
         {
+            var laData = GetLocalAuthorityData();
+            var abc = laData.GroupBy(x => x.Country).ToList();
+
             return View(ViewNames.LocalAuthorityDisposalCostsIndex);
         }
+
+        private List<LocalAuthorityViewModel> GetLocalAuthorityData()
+        {
+            var localAuthorityDisposalCosts = GetLocalAuthorityDisposalCosts();
+
+            var localAuthorityData = new List<LocalAuthorityViewModel>();
+
+            if (localAuthorityDisposalCosts.Any())
+            {
+                foreach (var la in localAuthorityDisposalCosts)
+                {
+                    localAuthorityData.Add(new LocalAuthorityViewModel(la));
+                }
+            }
+
+            return localAuthorityData;
+        }
+
+        private List<LocalAuthorityDisposalCost> GetLocalAuthorityDisposalCosts()
+        {
+            var localAuthorityDisposalCosts = new List<LocalAuthorityDisposalCost>();
```

Comment:
This is more of a suggestion. Can you move this to a separate file called mock. You could use that in unit tests also. 
Does not look good where controller has the mock data. 
Thanks